sdk/resourcemanager/Azure.ResourceManager.Core/src/BasicContainerBase.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@m-nash, name on this base class? Let's discuss if we should move it into the class hierarchy tree of ContainerBase/ResourceContainerBase.